src/core/api/clock.ts

```diff
@@ -172,7 +181,8 @@ function hasLoadedUntilTheEnd(
  */
 function getMediaInfos(
   mediaElement : HTMLMediaElement,
-  currentState : IMediaInfosState
+  currentState : IMediaInfosState,
+  isInternalSeekingComing = false
```

Comment:
Needs some comments here and where the initial variable is declared to explain why this is needed

Reply:
I added comments to explain it